webofneeds/won-owner-webapp/src/main/java/won/owner/web/websocket/WonHandshakeInterceptor.java

```diff
@@ -16,26 +16,32 @@
 
 package won.owner.web.websocket;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.http.HttpSession;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.http.server.ServerHttpRequest;
 import org.springframework.http.server.ServerHttpResponse;
 import org.springframework.http.server.ServletServerHttpRequest;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.web.socket.WebSocketHandler;
 import org.springframework.web.socket.server.support.HttpSessionHandshakeInterceptor;
 
-import javax.servlet.http.HttpSession;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-
 /**
  * User: LEIH-NB
  * Date: 09.10.2014
  */
 public class WonHandshakeInterceptor  extends HttpSessionHandshakeInterceptor
 {
+
+	private final Logger logger = LoggerFactory.getLogger(getClass());
+	
   // TODO why do we need this if there is a session cookie?
-  public static final String SESSION_ATTR = "httpSession.id";
+  public static final String SESSION_ATTR = "HTTP.SESSION.ID";
   // TODO  probably we don't need it any more, the user in WonWebSocketHandler
```

Comment:
do we need the code? can we delete the comments then?

Reply:
We need it. Let's delete the comments